wire-library/settings.gradle.kts

```diff
@@ -23,3 +23,11 @@ if (startParameter.projectProperties.get("swift") != "false") {
   include(":wire-tests-swift")
   include(":wire-tests-proto3-swift")
 }
+
+dependencyResolutionManagement {
+  versionCatalogs {
+    create("libs").from(files("../gradle/libs.versions.toml"))
```

Comment:
I guess you can do `create("deps")` to avoid having to change build files?

Reply:
I don't understand what you're suggesting? I wanna get rid of the `Dependencies.kt` file